### ISSUE TITLE
Encoding - German Umlaute Umlauts

### ISSUE THREAD

**Issue by SirTomme** [openhab/openhab.pebble#38]
Displaying the German Umlauts (äöüÄÖÜ) on a sitemap does not seem to be possible with the pebble app. Tried different encodings without success: UTF-8, UTF-16, Latin 1, Mac Roman, Windows.

**Comment by llamahunter** [comment]
Hmm. That's odd. I'll look into it when I get back into town in a few weeks. What happens? Are they swallowed? Render as a box? Crash the app?

**Comment by SirTomme** [comment]
No, not swallowed, no crash. Some Examples how stuff is being rendered when the sitemap is utf-8:
Ä ==> Ã▯
ü ==> Ã¼
° ==> Â°  // degree has problems too

Some more information from my watch:
Model: Pebble Time Round
Firmware: v4.3
Language: de_DE, v34 (the problem persists when changing to en_US, v1)
Thanks in advance!
